deps/readies/bin/getpy

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+show_if_error() {
+	{ "${@:1}"; } > /tmp/py.log  2>&1
+	[ $? != 0 ] && cat /tmp/py.log
+	rm -f /tmp/py.log
+}
+
+if [ ! -z $(command -v python) ]; then
+	[ "$(python --version 2>&1 | cut -d" " -f2 | cut -d. -f1)" = "3" ] && exit 0
+fi
+
+[ ! -z $(command -v python3) ] && exit 0
+
+if [ ! -z $(command -v apt-get) ]; then
+	show_if_error apt-get -qq update
```

Comment:
As a "good citizenship" suggestion: in case any system-wide installs are needed `readies` should stop and ask "Should I install this and this? [Yn]", and possibly have a `-y` flag that overrides that prompt (for Docker builds etc)

Reply:
The system-setup script has a dual functionality: it can set up a machine to successfully build, run, and test the module, and also provide documentation of what the system requirements are, for a bloke that's vigilant about his system diet.
It is possible to add a safety question feature easily overrided in docker environments (like ASK=0 ./getpy). I'll add that.
There also the --nop option, that is similar to make's -n.
There is another aspect that I'm considering: having a Python virtual environment for all pip installations. This will probably added across modules when I integrate it with the build, testing and packaging process. Nothing is easy in life. :-)